src/DIRAC/FrameworkSystem/scripts/dirac_login.py

```diff
@@ -271,7 +271,9 @@ def loginWithCertificate(self):
         # Load user cert and key
         result = chain.loadChainFromFile(self.certLoc)
         if result["OK"]:
-            result = chain.loadKeyFromFile(self.keyLoc, password=getpass.getpass("Enter Certificate password:"))
+            result = chain.loadKeyFromFile(
+                self.keyLoc, password=prompt(u"Enter Certificate password: ", is_password=True)
```

Comment:
As the `u"` doesn't do anything in Python 3:
```suggestion
                self.keyLoc, password=prompt("Enter Certificate password: ", is_password=True)
```